pom.xml

```diff
@@ -536,6 +543,19 @@ under the License.
 				</plugins>
 			</build>
 		</profile>
+
+		<profile>
+			<activation>
+				<activeByDefault>false</activeByDefault>
+				<property>
+					<name>asyncProfilerLib</name>
+				</property>
+			</activation>
+			<id>enable-async-profiler</id>
+			<properties>
+				<jmhProfArgument>async:libPath=${asyncProfilerLib};output=flamegraph;dir=/tmp/profile-results</jmhProfArgument>
```

Comment:
`libPath` is user and platform specific so I don't think we can sensibly default it.